content/en/posts/mesa-optimizers.md

```diff
@@ -0,0 +1,40 @@
+
+However, the training loop is not some omniscient or omnipotent entity that just happens to find the best possible program or agent. It is still a relatively simple process, that usually starts with some randomized initialization of model weights, and then performs a greedy search process (such as SGD), iteratively improving the model weights. It's unclear under which circumstances exactly such a process would actually *find* a set of model weights that would yield a mesa optimizer for any given initialization.
+
+Yet, despite these uncertainties, there are at least two reasons to take this idea quite seriously:
+
+1. Even if currently training loops mostly don't end up creating mesa optimizers and maybe even lack the capabilities to identify such programs in the vast multi-dimensional search space of model weights, we just need to look one level up: behind each ML training process, there's a bunch of **human researchers** who constantly tweak the training regime in order to come up with the most capable possible models. These humans **have an incentive to keep adjusting the training process until it will eventually be able to create mesa optimizers**.
+2. **Humans could be considered exemplary for mesa optimization**: a simple outside process, namely evolution, has run an optimization on living beings, selecting inclusive genetic fitness, and eventually (quite misaligned, from the point of view of evolution) optimizers emerged in the form of humans.
+
+One caveat to all of this is that the concept of an "optimizer" is difficult to define cleanly. And while it's easy to think of optimizer-ness as a binary property, it's probably more continuous, with some AIs being more and others less "optimizer-like". Looking at the example of humans mentioned above, it's questionable to what degree humans really are optimizers. Some individuals are optimizing for their goals much more than others, but practically all humans are investing some effort to shape the world in accordance with their preferences. So while many humans exhibit optimizer-like behaviour, we're certainly far from *perfect* optimizers.
+
+If we do eventually end up with advanced optimizer AIs that strongly exceed human abilities, then it's certainly appealing to follow e.g. Bostrom's arguments about instrumental convergence or the orthogonality thesis, and become rather concerned about the safety of such AIs.
```

Comment:
This paragraph is a little wishy washy for my taste. "certainly appealing" and "rather concerned" are vague. I would remove the paragraph.

Reply:
Will probably add this to the end of the previous paragraph instead, which hopefully is less wishy-washy without making any over-confident speculative claims: "But whatever may be true for humans, it's easy to see that if we end up with some very capable mesa optimizer AIs, which are not properly aligned with our goals, this can lead to some very undesirable outcomes."